tests/expressions/tests.py

```diff
@@ -332,12 +336,195 @@ def test_object_update_unsaved_objects(self):
         with self.assertRaisesMessage(ValueError, msg):
             acme.save()
 
+    def test_ticket_22288_in_f_integer(self):
+        # Test that in lookups can use F() expressions for integers
+        ceo = Employee.objects.create(firstname="Just", lastname="Doit", salary=30)
+        # MySQL requires that the values calculated for expressions do not
+        # pass outside of the field's range. We delete the standard company
+        # objects as they have inconvenient values for these tests
+        Company.objects.all().delete()
```

Comment:
Maybe just put these tests in a separate TestCase to avoid useless creation and deletion of companies? Especially since you have two tests doing identical setup of alternate company data.